tutorials/frontend/deploy_object_detection_pytorch.py

```diff
@@ -27,16 +27,16 @@
 
 .. code-block:: bash
 
-    pip install torch==1.4.0
-    pip install torchvision==0.5.0
+    pip install torch==1.7.0
+    pip install torchvision==0.8.1
 
 or please refer to official site
 https://pytorch.org/get-started/locally/
 
 PyTorch versions should be backwards compatible but should be used
 with the proper TorchVision version.
 
-Currently, TVM supports PyTorch 1.4 and 1.3. Other versions may
+Currently, TVM supports PyTorch 1.7 and 1.4. Other versions may
```

Comment:
nit: is that ok to claim 1.4 support even if we don't test with it? (please correct me if this doesn't make sense)

Reply:
it's a good point, my reasoning is that since we've been testing with v1.4, and this upgrade doesn't break it (pytorch frontend tests pass on both 1.4 and 1.7), I think we can claim we still support 1.4.